src/main/java/de/unijena/cheminf/fragment/fingerprint/FragmentFingerprinter.java

```diff
@@ -153,22 +113,16 @@ public FragmentFingerprinter(List<String> aFragmentList) throws NullPointerExcep
      */
```

Comment:
In the doc again, NullPointerExc. is thrown if the list *or any of its elements* is null.